androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/StopDetailsFilteredDeparturesView.kt

```diff
@@ -79,6 +79,7 @@ fun StopDetailsFilteredDeparturesView(
     data: FilteredDeparturesData,
     tileData: List<TileData>,
     noPredictionsStatus: NoTripsFormat?,
+    hidePredictions: Boolean,
```

Comment:
suggestion (non-blocking): Consider defaulting to false here to avoid having to set it in all unaffected tests